core/src/v2/actix.rs

```diff
@@ -568,7 +568,7 @@ macro_rules! impl_path_tuple ({ $($ty:ident),+ } => {
                         description: v.description,
                         collection_format: None, // this defaults to csv
                         items: v.items.as_deref().map(map_schema_to_items),
-                        name: k,
+                        name: String::new(),
```

Comment:
I don't know the code very well at all. Does it matter at all that `k` is used for determining whether the parameter is "required", but then discarding it when determining the parameter "name"?  Is it possible we end up marking the wrong parameters as "required"?

Reply:
Not here, this is the impl for path tuples: Path<($($ty,)+)
So here the names are just indexes, "0", "1"....

But even with individual paths, so two Path<>, Path<>, the names are not matched with the url names, example:
/{continent}/{country}
fn f(country: Path<String>, continent: Path<String>)...

Here as you see, country and continent are swapped, but no error is thrown, in fact the arg names in the function seem to be completely ignored..
Fixing this will require a lot more changes probably.